storagenode/piecestore/endpoint.go

```diff
@@ -125,6 +125,7 @@ func (endpoint *Endpoint) Delete(ctx context.Context, delete *pb.PieceDeleteRequ
 func (endpoint *Endpoint) Upload(stream pb.Piecestore_UploadServer) (err error) {
 	ctx := stream.Context()
 	defer mon.Task()(&ctx)(&err)
+	startTime := time.Now()
```

Comment:
I agree with Stefan that this and other places using time.Now() should probably be time.Now().UTC() for consistency